src/app/api/product-reviews/user/[userId]/route.ts

```diff
@@ -0,0 +1,41 @@
+import { getProductsReviewsByUser } from "@/model/ProductReviewsModel";
+import { NextResponse } from "next/server";
+
+export async function GET(
+  request: Request,
+  { params }: { params: { userId: string } }
+) {
+  const { userId } = params;
+  try {
+    let reviews = await getProductsReviewsByUser(userId);
+    if (reviews !== undefined) {
```

Comment:
This `if` will never happen. If you look at the `getProductsReviewsByUser` function, it will always return an array.

So actually we can remove this `if`. If there are no reviews, the response will be an empty array, which is fine.